systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterIT.java

```diff
@@ -342,4 +344,44 @@ public void testJvmAndResources() {
         assertResources(NAMESPACE, podName,
                 "500M", "300m", "500M", "300m");
     }
+
+    @Test
+    @JUnitGroup(name = "regression")
+    @KafkaCluster(name = CLUSTER_NAME, kafkaNodes = 3, config = {
+            @CmData(key = "kafka-config", value = "{\"default.replication.factor\": 3,\"offsets.topic.replication.factor\": 3,\"transaction.state.log.replication.factor\": 3}")
+    })
+    @Topic(name = TOPIC_NAME, clusterName = "my-cluster")
```

Comment:
What's the reason that you created a new topic in pre-conditions and then don't use this topic in system tests? Maybe you should delete this pre-condition or maybe you can remove creating "test-topic1" in test method?  WDYT?
As I understood by description to this PR you wanna do `creation/update/deletion` for the `topic` in several ways but in lines 358-359 you create the topics only by CLI. So maybe the right solution will create one of them by `oc`? Like you do by using annotation `@Topic`